stock_location_lockdown/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Location Lockdown",
+    "summary": "Prevent to add stock on locked locations",
```

Comment:
@MeetKD I've just changed it. Is the rest of the module ok?

Reply:
Code Review :+1: